packages/fluent-theme/src/components/activity/ActivityDecorator.module.css

```diff
@@ -0,0 +1,301 @@
+
+  &:hover {
+    cursor: pointer;
+    color: var(--webchat-colorBrandForeground2Hover);
+    border-color: var(--webchat-colorBrandStroke2Hover);
+    background-color: var(--webchat-colorBrandBackground2Hover);
+  }
+  
+  &:hover:active {
+    color: var(--webchat-colorBrandForeground2Pressed);
+    border-color: var(--webchat-colorBrandStroke2Pressed);
+    background-color: var(--webchat-colorBrandBackground2Pressed);
+  }
+
+  &::before, &::after {
+    all: unset;
+  }
+
+  &:has(:global(.webchat__render-markdown__external-link-icon)) {
+    padding-inline: 3px;
+  }
+
+  :global(.webchat__render-markdown__external-link-icon) {
+    background: currentColor;
+    -webkit-mask: var(--webchat__icon-url--external-link) no-repeat;
+    mask: var(--webchat__icon-url--external-link) no-repeat;
+    height: 0.7em;
+  }
+}
+
+/* Citation summary chevron */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__link-definitions__header) {
+  :global(.webchat__link-definitions__header-text) {
+    color: var(--webchat-colorNeutralForeground3);
+  }
+
+  :global(.webchat__link-definitions__header-chevron) {
+    fill: var(--webchat-colorNeutralForeground3);
+    font-size: var(--webchat__font-size--small);
+    width: 1em;
+  }
+}
+
+/* Citation link definitions */
+:global(.webchat-fluent) .activity-decorator :global(.webchat__link-definitions) {
+  --webchat__citation__external-link--mask: var(--webchat-externalLink-mask);
+  --webchat__citation__link--max-width: var(--webchat-externalLink-maxWidth);
+
+  :global(.webchat__link-definitions__list) {
+    color: var(--webchat__color--subtle);
+    display: flex;
+    flex-flow: row wrap;
+    gap: var(--webchat-spacingHorizontalS);
+  }
+
+  :global(.webchat__link-definitions__list-item) {
+    max-width: var(--webchat__citation__link--max-width);
+    border-radius: var(--webchat-borderRadiusMedium);
+  }
+
+  :global(.webchat__link-definitions__list-item-box) {
+    background-color: var(--webchat-colorNeutralBackground3);
+    border-radius: var(--webchat-borderRadiusMedium);
+    border: var(--webchat-strokeWidthThin) solid var(--webchat-colorNeutralStroke2);
+    box-sizing: border-box;
+    color: currentColor;
+    display: inline-flex;
+    font-size: var(--webchat__font-size--small);
+    height: 24px;
+    padding-inline-end: var(--webchat-spacingHorizontalS);
+  }
+
+  :global(.webchat__link-definitions__list-item-body) {
+    font-family: inherit;
+    gap: var(--webchat-spacingHorizontalSNudge);
+    min-width: 0;
+    padding: 0;
+  }
+
+  :global(.webchat__link-definitions__badge) {
+    align-self: center;
+    background-color: transparent;
+    border-radius: 0;
+    border: none;
+    color: currentColor;
+    font-size: var(--webchat-fontSizeBase100);
+    font-weight: var(--webchat-fontWeightSemibold);
+    line-height: var(--webchat-lineHeightBase100);
+    margin: 0;
+    padding: 0;
+    position: relative;
+    text-align: center;
+    width: 20px;
```

Comment:
Will `20px` too restrictive? Should it be `min-width` instead?

Reply:
Ripped from Teams, I should've check it more. Right.